### ISSUE TITLE
package not registered with npmjs yet?

### ISSUE THREAD

**Issue by ahmdsk** [SteveYuOWO/vue-hot-toast#5]

**Comment by SteveYuOWO** [comment]
Hi, ahmdsk. Was the url accessed incorrectly? @ahmdsk 
https://www.npmjs.com/package/@steveyuowo/vue-hot-toast

**Comment by roberto-slopez** [comment]
I tried to install it but...
```cmd
$: npm i @steveyuowo/vue-hot-toast --save
npm ERR! code EUNSUPPORTEDPROTOCOL
npm ERR! Unsupported URL Type "link:": link:..
```

**Comment by SteveYuOWO** [comment]
@roberto-slopez Sorry for the late reply

https://github.com/SteveYuOWO/vue-hot-toast/commit/969cbe626ceb20d516622a69ee551d72959972b4#diff-7ae45ad102eab3b6d7e7896acd08c427a9b25b346470d7bc6507b6481575d519L29

This bug was fixed in v1.2.0
